client/state/rewind/selectors/get-backup-retention-update-status.ts

```diff
@@ -0,0 +1,16 @@
+import { BACKUP_RETENTION_UPDATE_REQUEST } from '../retention/constants';
+import type { AppState } from 'calypso/types';
+import 'calypso/state/rewind/init';
+
+/**
+ * Returns the update backup retention request's status.
+ *
+ * @param state The application state.
+ * @param siteId The site ID for which to retrieve days of backups saved.
+ * @returns The the status of the request.
+ */
+const getBackupRetentionUpdateRequestStatus = ( state: AppState, siteId: number ): string =>
+	state.rewind[ siteId ]?.updateBackupRetentionRequestStatus ??
```

Comment:
Considering that the reducer has a default value, `updateBackupRetentionRequestStatus` should be set to `BACKUP_RETENTION_UPDATE_REQUEST.UNSUBMITTED` by default right? So the nullish coalescing is likely unnecessary.

Reply:
Correct! But the unit tests were failing since they run isolated - when testing `getBackupRetentionUpdateRequestStatus` the first case fails and returns `undefined` instead of `UNSUBMITTED`, browser console does show `UNSUBMITTED` as default.

Reply:
Aren't the unit tests supposed to reflect the actual real use case and behavior?

But I do see some rationale in making it tighter 😉 I don't mind keeping the nullish coalescing, just wanted to point it out.